netsuitesdk/api/expense_reports.py

```diff
@@ -97,5 +100,6 @@ def post(self, data) -> OrderedDict:
             er['entity'] = self.ns_client.RecordRef(**(data['entity']))
 
         logger.debug('able to create er = %s', er)
-        res = self.ns_client.upsert(er)
+        res = self.ns_client.upsert(er,  'er')
```

Comment:
lets use full name instead of short forms
```suggestion
        res = self.ns_client.upsert(er,  'expense_report')
```